agent/githubHook.py

```diff
@@ -59,4 +59,4 @@ def checkout_pr(pr):
                 f.write(auth.token + '\r\n')
                 f.write(str(auth.id))
         except Exception as e:
-            print e
+            print e.response.json()
```

Comment:
The original `Exception` does not contain the `response` object.
It will raise another Exception when `e` is not `GitHubError`, for example, file-related exceptions.